app/src/main/java/com/korneysoft/rsschool2021_android_task_6_musicapp/viewmodel/MainViewModel.kt

```diff
@@ -0,0 +1,60 @@
+package com.korneysoft.rsschool2021_android_task_6_musicapp.viewmodel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import com.korneysoft.rsschool2021_android_task_6_musicapp.data.Track
+import com.korneysoft.rsschool2021_android_task_6_musicapp.music_service.ServiceConnectionController
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private const val TAG = "T6-MainViewModel"
+
+@Singleton
```

Comment:
We need annotation @Singleton in ViewModel, because if don't use it will be created new instance ViewModel  every time when will be call @Inject (while creating Activity).
Or we must use "private val viewModel: MainViewModel by viewModels()" in Activity instead of @Inject